api/handlers/handlers.go

```diff
@@ -31,6 +31,9 @@ func API(log *log.Logger) http.Handler {
 
 	//groups
 	gH := groupsHandler{log}
+	router.HandleFunc("/groups/{env}", gH.list).
+		Methods("GET").Queries("parent", "{parent:[01]}")
```

Comment:
Is it `parent` or `parents`?
```suggestion
	router.HandleFunc("/groups/{env}", gH.list).
		Methods("GET").Queries("parents", "{parents:[01]}")
```

If you call `groups/{env}` without `?parent` does it still works?

Reply:
yes since there are 2 routes for "/groups/{env}" with or without "parents" param

> Is it parent or parents?
ok go with the S